### PR TITLE
feat(profile): add `profile` subcommands

### PR DESCRIPTION
**Pull request by alopezsanchez**
Ref: https://www.npmjs.com/package/npm-profile
Ref: https://docs.npmjs.com/cli/profile.html
Ref: https://docs.npmjs.com/cli/token.html

- [ ] `adduser`
- [ ] `login`
- [ ] `adduserWeb`
- [ ] `loginWeb`
- [ ] `adduserCouch`
- [ ] `loginCouch`
- [x] `get`
- [x] `set`
  - [x] `enable-2fa`
  - [x] `disable-2fa`
  - [x] `set password`
- [x] `listTokens`
- [x] `removeToken`
- [x] `createToken`

### CODE REVIEW

**Comment by alopezsanchez** [comment]
Hello @zkat, I have some questions.
* I'm not sure how to implement `login*` and `adduser*` subcommands. I don't understand the `opener` and `prompter` arguments.
* Should we add the three types of login/adduser?

Thanks!

**Comment by alopezsanchez** [comment]
@zkat If you think that `login` and `adduser` subcommands are out of scope, I think you can review this! 😃 

Btw, I have some improvements in my TODO list:
* Tests
* Improve CLI UX with `ink`
* Improve `tink profile get` table
* Logging

File: lib/commands/profile.jsx [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+      console.log('Two factor authentication disabled.')
+    }
+  } catch (e) {
+    logError(e)
+  }
+}
+
+async function enable2fa (argv) {
+  const url = require('url')
+  const queryString = require('query-string')
```
Comment (zkat) [inline lib/commands/profile.jsx]: Why not use the built-in `querystring` module?

File: lib/commands/profile.jsx [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+    describe: 'Used for Basic auth',
+    type: 'string'
+  }
+}
+
+const tableHeaders = ['token', 'key', 'cidr_whitelist', 'readonly', 'created', 'updated']
+
+const getOptions = argv => ProfileConfig(npmConfig().concat(argv).concat({ log }))
+
+// TODO: Move to `../utils`?
```
Comment (zkat) [inline lib/commands/profile.jsx]: Nah

File: lib/commands/profile.jsx (hunk 1 above)
Comment (alopezsanchez) [reply lib/commands/profile.jsx]: You are right. Changed 👍

File: lib/commands/profile.jsx (hunk 2 above)
Comment (alopezsanchez) [reply lib/commands/profile.jsx]: OK!

**Review by zkat — APPROVED** [review]
So far so good!

**Pull request merged**